recipes/flufl.enum/meta.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: flufl.enum
+  version: "4.1.1"
```

Comment:
Take a look at the example and switch the 4.1.1 in various places to the jinja variable to improve the recipe maintainability later on.